### PR TITLE
Translate documentation files (PT-BR): Utility Types / Variable Declarations

### PR DESCRIPTION
**Pull request by Perkles**
This PR (Ref #233 ) translates the following files to Brazilian Portuguese:

Utility Types 
Variable Declarations

### CODE REVIEW

**Comment by Perkles** [comment]
Hey @khaosdoctor  👋 
Você poderia me auxiliar com esse erro no CI ?

File: packages/documentation/copy/pt/reference/Utility Types.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+
+type Pagina = "inicio" | "sobre" | "contato";
+
+const nav: Record<Pagina, InfoPagina> = {
+  sobre: { titulo: "sobre" },
+  contato: { titulo: "contato" },
+  home: { titulo: "inicio" },
+};
+
+nav.about;
```
Comment (orta) [inline packages/documentation/copy/pt/reference/Utility Types.md]: You left `home` and `about` in this example, which is what the compiler is complaining about. I believe you want:

```suggestion
  inicio: { titulo: "inicio" },
};

nav.sobre;
```

Comment (Perkles) [reply packages/documentation/copy/pt/reference/Utility Types.md]: That's it ! Thanks @orta 😄

**Comment by Perkles** [comment]
I'll make sure all code examples compile before commit.

**Comment by orta** [comment]
Good luck - you can run `yarn workspace documentation lint` to run the compiler over them

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**